main/zoe/api/contract-support/price-authority.md

```diff
@@ -136,3 +136,68 @@ const quote = E(pAuthority).quoteWhenLTE(
     amountMath.make(brands.Out, 974n),
   );
 ```
+## `MutableQuote` object
+
+**tyg todo: When do you want to use a mutable quote vs. a regular quote, and vice versa?**
```

Comment:
Use a MutableQuote when you expect to make multiple calls, replacing the trigger value. If you just need a single quote, and won't change the trigger level, then quoteWhen will work fine.